testsuite/pytests/test_aeif_cond_beta_multisynapse.py

```diff
@@ -0,0 +1,419 @@
+        vm = nest.Create("voltmeter", params={"interval": dt})
+
+        # Create one aeif_conf_beta_multisynapse neuron
+        multisynapse_neuron = nest.Create(
+            "aeif_cond_beta_multisynapse",
+            params={
+                "E_rev": E_rev,
+                "tau_decay": tau_decay,
+                "tau_rise": tau_rise,
+                "V_peak": V_peak,
+                "a": a,
+                "b": b,
+                "E_L": Vrest,
+                "g_L": g_L,
+            },
+        )
+
+        # create an array of synaptic indexes
+        synapses_idx = range(len(weights))
+
+        # connect spike generator to each port
+        for i in synapses_idx:
+            nest.Connect(
+                sg, multisynapse_neuron, syn_spec={"delay": delays[i], "weight": weights[i], "receptor_type": i + 1}
+            )
+
+        # connect voltmeter
+        nest.Connect(vm, multisynapse_neuron)
+
+        # run simulation
+        nest.Simulate(simulation_t)
+
+        # Get the membrane voltage of the multisynapse neuron
+        ts = vm.events["times"]
+        Vms = vm.events["V_m"]
+
+        if have_plotting:
+            # plot timeseries as a sanity check
+            import matplotlib.pyplot as plt
+
+            fig, ax = plt.subplots(nrows=6)
+            ax[0].plot(ts, Vms, label="V_m multisyn", alpha=0.5)
+
+        V_m_summed = 0.0
+
+        # loop over synapse index
+        for i in range(3):
+            t0 = spike_time + delays[i]  # peak starting time
+
+            # Evaluates the driving force
+            Vforce = E_rev[i] - Vrest
+
+            # coefficient for approximate linear relationship between V(t) and g(t)
+            coeff = weights[i] * Vforce / g_L
+
+            # peak time
+            t_p = tau_decay[i] * tau_rise[i] / (tau_decay[i] - tau_rise[i]) * np.log(tau_decay[i] / tau_rise[i])
+
+            # normalization coefficient for synaptic conductance
+            g0 = coeff / (np.exp(-t_p / tau_decay[i]) - np.exp(-t_p / tau_rise[i]))
+
+            # approximate analytical calculation of V(t)
+            Vtheor = g0 * (np.exp(-(ts - t0) / tau_decay[i]) - np.exp(-(ts - t0) / tau_rise[i]))
+            Vtheor[ts < t0] = 0.0
+            V_m_summed += Vtheor
+            Vtheor += Vrest
+
+            if have_plotting:
+                ax[i + 1].plot(ts, Vtheor)
+
+                for _ax in ax:
+                    _ax.legend()
+
+        V_m_summed += Vrest
+
+        if have_plotting:
+            ax[0].plot(ts, V_m_summed, label="summed")
+
+            ax[-1].semilogy(ts, np.abs(Vms - V_m_summed), label="error")
+            fname = "test_aeif_cond_beta_multisynapse.png"
+            fname = os.path.join(os.environ.get("REPORTDIR", ""), fname)
+            fig.savefig(fname)
+
+        # large testing tolerance due to approximation (see documentation of the test)
+        np.testing.assert_allclose(Vms, V_m_summed, atol=0.0, rtol=1e-5)
```

Comment:
The convention in other tests have been `import numpy.testing as nptest`. It might be good to have this here also for conformity.

Reply:
I would prefer ``numpy.testing``, it's more descriptive and less ambiguous.

Reply:
I'd suggest to stick to convention in this PR `import numpy.testing as nptest`, and then consistently switch all tests in a separate PR if everyone agrees.

Reply:
```
$ grep -r "np\.testing\.assert" testsuite/  | wc -l
42
$ grep -r "nptest\.assert" testsuite/  | wc -l
70
```
I would suggest to leave as-is in this PR and do this consistently for all tests in a separate PR.